src/h5web/visualizations/heatmap/MappedHeatmapVis.tsx

```diff
@@ -57,6 +70,8 @@ function MappedHeatmapVis(props: Props): ReactElement {
       scaleType={scaleType}
       keepAspectRatio={keepAspectRatio}
       showGrid={showGrid}
+      abscissaParams={{ label: abscissaLabel, values: abscissas as number[] }}
+      ordinateParams={{ label: ordinateLabel, values: ordinates as number[] }}
```

Comment:
These `as number[]` have been bothering me, but I assume TS is being annoying... It's strange that it can't infer the type from the conditional assertion... Perhaps `assertOptionalArray<number>` would solve it?

Reply:
> These `as number[]` have been bothering me, but I assume TS is being annoying... It's strange that it can't infer the type from the conditional assertion...

That is exactly that.
> Perhaps `assertOptionalArray<number>` would solve it?

Good idea, I will try that